bmp_mode_snake.cc

```diff
@@ -2,10 +2,13 @@
 #include <cstdint>
 #include <cstring>
 
+#include <stdlib.h>
```

Comment:
prefer <cstdlib> to <stdlib.h> and that general form in c++ for these [reasons](https://stackoverflow.com/a/301907)